src/utils/constants.ts

```diff
@@ -126,26 +136,14 @@ export const SubscriptionFields = {
     NewSub_OptionLabel: 'Create New',
     NewSub_OptionValue: 'newsubscription',
 
+    PrefixCustomDefinitionSubject: 'custom_fields_',
     PrefixTriggersTitle: '__mm_webhook',
     RegexTriggerInstance: '__instance_',
     RegexTriggerChannelID: '__channelID_',
-    RegexTriggerTitle: '__mm_webhook__instance_(.*)__channelID_(\\w+) (.*)',
-
-    // TODO add ticket is created and ticket is updated options
-    ConditionsCheckBoxFields: [
-        'status',
-        'priority',
-        'brand',
-        'form',
-        'type',
-        'group',
-        'assignee',
-        'requester',
-        'organization',
-    ],
+    RegexTriggerTeamID: '__teamID_',
+    RegexTriggerTitle: '__mm_webhook__instance_(.*)__teamID_(\\w+)__channelID_(\\w+) (.*)',
```

Comment:
I wonder if you can make the trigger name in a URL param fashion like:

`My trigger title?instance=(instance)&teamID=(team id)&channelID=(channel id)`